manifests/m/Mozilla/FirefoxNightly/91.0a1/Mozilla.FirefoxNightly.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate
+
+PackageIdentifier: Mozilla.FirefoxNightly
+PackageVersion: 91.0a1
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/nightly/latest-mozilla-central/firefox-91.0a1.en-US.win64.installer.exe
+  InstallerSha256: 07AE0F404249857D2259219B09771724805B2BCFFAC38A86B537B69B7E64ECB6

```

Comment:
```
  InstallerSha256: 07AE0F404249857D2259219B09771724805B2BCFFAC38A86B537B69B7E64ECB6
```
The hash will need to be updated because this is a nightly build and the hash will always change every day:
![image](https://user-images.githubusercontent.com/11600822/122371715-fab1e900-cf57-11eb-9f12-e18ce6e8ead7.png)